### DISCUSSION TITLE
[Not a bug] Renaming my bucket named "utils"

### DISCUSSION THREAD

**Discussion by couleurm** [ScoopInstaller/Scoop#5212] [Help and Questions]
I'm unsure where/what to label this issue sorry

My [bucket](https://github.com/couleur-tweak-tips/utils) has the very generic name "`utils`" and I'd like to rename it to the name of my org [`ctt`](https://github.com/couleur-tweak-tips/), but that would break all users using my bucket, what options do I have?

* `git push --mirror`
* `Write-Host $NewBucketWarning;scoop bucket add newbucket;exit` in all my manifest's JSON pre_install keys (a silly one indeed)

(I'd also like having my bucket in [scoop.sh](http://scoop.sh)'s cache and I don't think I'll get in with such name 😋)

**Comment by niheaven** [comment]
The bucket name is written into `install.json`, and you could check the file, and use utils such as `grep` or `sed` to replace all of them.

**Comment by Ryanjiena** [comment]
```powershell
scoop update *
$scoop_dir = "D:\Scoop"
$apps = (scoop status | Where-Object Info -Match ".*Manifest removed").Name
Write-Output $apps | ForEach-Object { (Get-Content "$scoop_dir\apps\$_\current\install.json") -replace 'oldbucket', 'newbucket' | Set-Content "$scoop_dir\apps\$_\current\install.json" -Encoding ASCII }
```
